<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lslVisual</class>
 <widget class="QMainWindow" name="lslVisual">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1142</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LSL Visualization</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="queryButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Query</string>
    </property>
   </widget>
   <widget class="QPushButton" name="visualizeButton">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>90</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Visualize Timeseries</string>
    </property>
   </widget>
   <widget class="QLabel" name="UTKlogo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="NBL">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nonlinear Biodynamics Lab</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>841</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="TimeSeriesViewer"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>640</y>
      <width>831</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="streamData"/>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>929</x>
      <y>600</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="applyFilter"/>
   </widget>
   <widget class="QPushButton" name="visualizeTimeFreqButton">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>90</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Visualize Timefrequency</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>700</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="resumeView"/>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>700</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="pauseView"/>
   </widget>
   <widget class="QScrollArea" name="channelLayout">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>150</y>
      <width>301</width>
      <height>181</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>299</width>
       <height>179</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="queryStream">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>150</y>
      <width>761</width>
      <height>181</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>759</width>
       <height>179</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>340</y>
      <width>711</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="TimeFrequencyViewer"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>929</x>
      <y>340</y>
      <width>191</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="filterLayout"/>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Available Streams</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>130</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Available Channels</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1142</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="LSL_Visualization">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>LSL Visualization</string>
    </property>
   </widget>
   <addaction name="LSL_Visualization"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
